app/src/main/java/com/duckduckgo/app/browser/di/BrowserModule.kt

```diff
@@ -374,6 +378,11 @@ class BrowserModule {
         return AndroidFileDownloader(dataUriDownloader, networkFileDownloader)
     }
 
+    @Provides
+    fun printInjector(): PrintInjector {
+        return PrintInjectorJS()
+    }
+
```

Comment:
nit: I know we have a mixture of patterns here, ie. sometimes we use provider methods in dagger modules, sometimes we directly annotated the provided class.
I personally prefer the later, eg.
```kotlin
@ContributesBinding(AppScope::class)
class PrintInjectorJS @Inject constructor() : PrintInjector {
}
```

My reasons for that are:
* just looking at the `PrintInjectorJS` class I can see how it is provided in DI
* tomorrow we need to add dependencies to `PrintInjectorJS` and we only need to modify one file
* dagger modules couple the dagger graph with the build graph

Reply:
I also prefer constructor injection, I made it through provides method to be consistency with other JS injectors but I will be happy to change it 🙂

Reply:
I finally changed it to constructor injection. I thought that if someone creates another JSInterface in the future, they can have this one as an example to start doing it like this. Thanks for the suggestion @aitorvs  💯